apps/ensadmin/package.json

```diff
@@ -43,6 +45,7 @@
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
     "date-fns": "^4.1.0",
+    "gqlpt": "0.0.0-alpha.31",
```

Comment:
hey @tk-o 

Thanks for the integration, it looks like the lib fit in nicely. We do have the alpha tag, your feedback and usage can really help us get to a stable position for the community. Feel free to open issues on gqlpt and or reach out directly ✌️